PyBASC/utils.py

```diff
@@ -285,16 +231,19 @@ def cross_cluster_timeseries(data1, data2, roi_mask_nparray, n_clusters, similar
         This is the matrix to receive cluster assignment
     data2 : array_like
         A matrix of shape (`N`, `M`) with `N2` samples and `M2` dimensions.
-        This is the matrix with which distances will be calculated to assign clusters to data1
+        This is the matrix with which distances will be calculated to assign
+        clusters to data1
     n_clusters : integer
         Number of clusters
-    similarity_metric : {'euclidean', 'correlation', 'minkowski', 'cityblock', 'seuclidean'}
-        Type of similarity measure for distance matrix.  The pairwise similarity measure
-        specifies the edges of the similarity graph. 'data' option assumes X as the similarity
-        matrix and hence must be symmetric.  Default is kneighbors_graph [1]_ (forced to be
-        symmetric)
+    similarity_metric : {'euclidean', 'correlation', 'minkowski', 'cityblock',
```

Comment:
Unclear whether we should keep this- certain clustering techniques require specific euclidean